### ISSUE TITLE
Thrusters module properties list has inconsistent capitalization

### ISSUE THREAD

**Issue by QbranchFatherhood** [EDCD/coriolis#790]
The words "optimal" and "max" are lowercase in the list of Thruster module properties which is not consistent with the formatting of other info, nor with the properties of other core/optional/hardpoint modules which are consistently capitalized.

![image](https://github.com/user-attachments/assets/94350e1c-5ed7-460b-811b-4b5d956f7c59)

**Comment by alex-williams** [comment]
I cannot replicate this issue on a Windows PC using Chrome, Firefox, Opera or Edge browsers... please let me know the browser you're using to get this problem and I'll investigate further. If not, I'll close the issue.

**Comment by QbranchFatherhood** [comment]
The current version of Safari on iOS 18.

**Comment by QbranchFatherhood** [comment]
![image](https://github.com/user-attachments/assets/967d424a-5f91-4fa0-afc6-a35d1e15a29b)

Max fuel for the FSD is "max Fuel".

**Comment by alex-williams** [comment]
> The current version of Safari on iOS 18.
> 
> The discussion on the Discord revolved around this being a mobile-specific problem, and that the libraries for that are reportedly well out of date. So, testing on a PC likely wouldn't reveal anything.

Indeed, I've tested it on an Android device now too however and can't replicate the issue there in Chrome or Firefox either, so I'm leaning towards this being a Safari issue (It pops out most of the weird display bugs...) with the possibility that the React Libraries being out of date maybe being a contributing factor. Once I upgrade the libraries, I'll check back with you to see if it resolves the issue, but there isn't much else I can do if it's just a Safari bug.
